docs/details/develop-checks.md

```diff
@@ -1,266 +1,325 @@
     - _PERCONA_TEST_CHECKS_RESEND_INTERVAL=2s_ to define the frequency for sending the SA-based alerts to Alertmanager.
-
+ 
     ```
     docker run -p 80:80 -p 443:443 --name pmm-server \
-
+ 
     -e PMM_DEBUG=1 \
     -e PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml \
     -e PERCONA_TEST_CHECKS_DISABLE_START_DELAY=true
     -e PERCONA_TEST_CHECKS_RESEND_INTERVAL=2s \
     perconalab/pmm-server:dev-latest
     ```
-
-3.  Log in to Grafana (admin/admin) and enable STT in the settings: http://127.0.0.1/graph/d/pmm-settings/pmm-settings
-
-    ![!](../_images/Grafana.png)
-
-4.  Create _/srv/custom-checks.yml_ inside a Docker container with the content from the Security Advisor (Security Threat Tool) section above.
-
-5.  STT checks will run with a time interval defined via UI. You can see the result of running the advisor on the home dashboard:
-
+ 
+3.  Log in to Grafana with credentials **admin/admin**.
+ 
+4. Go to **Configuration > Settings > Advanced Settings** and enable **Advisors**. For PMM 2.26 and older this option is called **Security Threat Tool**.
+ 
+4.  Create _/srv/custom-checks.yml_ inside a Docker container with the content of your check.
+ 
+5.  The checks will run according to the time interval defined on the UI. You can see the result of running the check on the home dashboard:
+ 
     ![!](../_images/HomeDashboard.png)
-
+ 
 6.  Click on the number of failed checks to open the Failed Checks dashboard:
-
+ 
     ![!](../_images/FailedChecks.png)
-
-7.  Go into Docker container to output the logs of pmm-managed and read STT logs:
-
+ 
+7.  Go into Docker container to output the logs of pmm-managed and read check logs:
+ 
 ```
 # get inside the container
 docker exec -it pmm-server bash
 # print and watch the logs
 supervisorctl tail -f pmm-managed
+ 
 ```
+### Function signature for Advisor checks
+ 
+The function signature for PMM 2.27 and later can be **check_context** (docs, context), where docs are lists of docs (one list of dicts for each query). 
+
+### No query parameter for MongoDB advisors
+ Contrary to Security Checks (developed for PMM 2.26 and older) Advisor checks for PMM 2.27 and later do not yet support a query parameter for MongoDB.
+ 
+## Advisor checks versus security checks
+PMM 2.26 and older included a set of security checks grouped under the **Security Threat Tool** option.
+ 
+Starting with the 2.27 release PMM introduced new checks and grouped them into set of Advisors, according to the functionality and recommendations they provide.
+ 
+To reflect these changes, the old **Security Threat Tool** option in PMM 2.26 in earlier has been renamed to **Advisors** and the checks use a slightly different format.
+
+### Develop security checks for PMM 2.26 and older
+The query for Security Checks (developed for PMM 2.26 and older) can also contain a MongoDB query document (as a string with proper quoting).
```

Comment:
Nope, they cannot. Too many mentions of this difference which not exists). Since we couldn't use query in previous versions nor we can use it in the current one, let's reduce amount of mentioning this not-implemented-yet feature.